src/Controller/GUIController.py

```diff
@@ -71,30 +71,35 @@ def open_new_patient(self):
         if confirmation_dialog == QMessageBox.Yes:
             self.open_patient_window.emit()
 
-    def pyradiomics_handler(self):
+    def pyradiomics_handler(self, path, filepaths, hashed_path):
         """
         Sends signal to initiate pyradiomics analysis
         """
         if which('plastimatch') is not None:
-            if self.hashed_path == '':
+            if hashed_path == '':
                 confirm_pyradi = QMessageBox.information(self, "Confirmation",
                                                     "Are you sure you want to perform pyradiomics? "
                                                     "Once started the process cannot be terminated until it finishes.",
                                                     QMessageBox.Yes, QMessageBox.No)
                 if confirm_pyradi == QMessageBox.Yes:
-                    self.run_pyradiomics.emit(self.path, self.filepaths, self.hashed_path)
+                    self.run_pyradiomics.emit(path, filepaths, hashed_path)
                 if confirm_pyradi == QMessageBox.No:
                     pass
             else:
-                self.run_pyradiomics.emit(self.path, self.filepaths, self.hashed_path)
+                self.run_pyradiomics.emit(path, filepaths, hashed_path)
         else:
             exe_not_found = QMessageBox.information(self, "Error",
```

Comment:
And this is exactly where you want to put in a mention of ensuring that the PATH includes the directory where the executable is installed.